src/Chainweb/Pact/PactService.hs

```diff
@@ -856,16 +856,26 @@ execValidateBlock
     => BlockHeader
     -> PayloadData
     -> PactServiceM cas PayloadWithOutputs
-execValidateBlock currHeader plData =
-    -- TODO: are we actually validating the output hash here?
-    withBatch $ withCheckpointerRewind mb "execValidateBlock" $ \pdbenv -> do
-        !result <- playOneBlock currHeader plData pdbenv
-        return $! Save currHeader result
+execValidateBlock currHeader plData = do
+    -- KILLSWITCH: The logic here involving `txSilenceDates` is to be removed in
+    -- a future version of Chainweb. This rejects any non-genesis block which is
+    -- found to contain transactions.
+    --
+    now <- liftIO getCurrentTimeIntegral
```

Comment:
This should be the block creation time and not the current time. Once a block got included into the chain, payload validation must always yield the same result.